package.json

```diff
@@ -70,13 +70,15 @@
     "mdast-util-from-markdown": "^1.3.0",
     "mdast-util-frontmatter": "1.0.1",
     "mdast-util-gfm-table": "^1.0.7",
+    "mdast-util-gfm-task-list-item": "^2.0.0",
```

Comment:
I have not upgraded the mdast family of packages to v2. This (and the one below) brings a second copy of `mdast-util-from-markdown`. Can you please downgrade the package and test if it works as expected?

Reply:
Makes sense, will downgrade the version to 1.x